keyboards/leeku/l3_tkl/l3_tkl.h

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2021 MajorKoos <https://github.com/majorkoos>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define _NO KC_NO
+
+#define LAYOUT_TKL( \
```

Comment:
required to be `LAYOUT_` and then lowercase with underscores

```suggestion
#define LAYOUT_tkl( \
```

Reply:
Done.